.bazelrc

```diff
@@ -0,0 +1,41 @@
+try-import %workspace%/user.bazelrc
```

Comment:
Should this go in `.gitignore`?

Reply:
It already is. I added bazel stuff to the gitignore a couple years ago even though at the time adding build scripts was not wanted.